docs/release-notes/2.40.0.md

```diff
@@ -0,0 +1,99 @@
+
+Before introducing this feature, locating the running services on a node was cumbersome. However, with this new feature, you can effortlessly access a list of services running on a specific node and identify the node name where the service is being utilized. Additionally, the filters implemented make navigation a lot simpler.
+
+  ![!](../../_images/PMM_Inventory_service_node_relation.png)
+
+
+#### Cluster view for Inventory [Technical Preview]
+
+Understanding the structure of your inventory is crucial. With the release of PMM 2.40.0, we've introduced an experimental feature that categorizes all your Services by Cluster, making it easier for you to understand your inventory. By using the **Organize by Clusters** toggle, you can view a group of services as a single cluster. PMM utilizes the cluster label to display services that belong to the same cluster.
+
+  ![!image](../../_images/PMM_Inventory_cluster_view.png)
+
+For detailed information on this feature, see [documentation](https://docs.percona.com/percona-monitoring-and-management/details/dashboards/dashboard-inventory.html#cluster-view)
+ 
+
+### MongoDB backup monitoring 
+PMM-12384: Now, PMM not only allows you to perform MongoDB backups from the UI, but it also displays backup status metrics. Additionally, we are shipping the first version of the Alert Template to notify you of any failed backups.
+[IMAGE HERE]
+For the usage of this Template, please use Alert documentation (LINK)
+
+
+### Components Upgrade
+
+VictoriaMetrics has been upgraded [v1.93.4](https://docs.victoriametrics.com/CHANGELOG.html#v1934).
+
+
+## New Features
+
+- [PMM-9374](https://jira.percona.com/browse/PMM-9374) - Starting with PMM 2.40.0, you can now use an external VictoriaMetrics database for monitoring in PMM. This provides multiple benefits, including scalability, resource isolation, reduced load on the PMM server,  customization, etc.
+- [PMM-11963](https://jira.percona.com/browse/PMM-11963) - Starting with PMM 2.40.0, you can click on the link in the **Node Name** column to view the node on which a specific service is running.
+- [PMM-11148](https://jira.percona.com/browse/PMM-11148) - [Inventory]: We have redesigned the **Select service type** and **Add service** pages to make them more user-friendly and intuitive.
+- [PMM-11423](https://jira.percona.com/browse/PMM-11423) - [Inventory]: Starting with PMM 2.40.0, you can now edit service labels directly from the PMM UI without having to remove and re-add them.
+- [PMM-12378](https://jira.percona.com/browse/PMM-12378) - [Inventory] - We have introduced an experimental feature that categorizes all your Services by Cluster, making it easier for you to understand your inventory. By using the **Organize by Clusters** toggle, you can view a group of services as a single cluster. 
+- [PMM-12384](https://jira.percona.com/browse/PMM-12378) - [BE] Backup metrics and alert rule template
```

Comment:
I don't know what is the right ticket for this, but this one is not :)
Also, the description seems to not mention what has been done.

Reply:
Fixed.